cluster/src/main/java/org/apache/iotdb/cluster/query/reader/ClusterReaderFactory.java

```diff
@@ -337,7 +362,8 @@ private AbstractMultPointReader getMultSeriesReader(
           deviceMeasurements,
           partitionGroup,
           context,
-          ascending);
+          ascending,
+          requiredSlots);
```

Comment:
always NULL?

Reply:
I have checked the logic and remove useless parameters.